.gitignore

```diff
@@ -110,3 +110,6 @@ buck-out/
 
 # CocoaPods
 /ios/Pods/
+
+# Maestro
+maestro/env.js
```

Comment:
Put a leading slash here. `/maestro/env.js`. That makes it relative to the repository root.